### PR TITLE
chore: [IA-706] Show error when url is empty or malformed

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR shows an error component when one of these error types occurred:

- the uri to load is empty or malformed (**data** error)
- the webview raises an error about the loading of the uri  (**webview** error)

| **data** error  | **webview** error |
| ------------- | ------------- |
Preferences | Privacy

## How to test
- use the dev-server
- to generate a **data** error use one of these blocks

empty case
```typescript
 handleInternalLink(
      dispatch,
      `ioit://${UADONATION_ROUTES.WEBVIEW}?urlToLoad= `
    );
```
malformed case
```typescript
 handleInternalLink(
      dispatch,
      `ioit://${UADONATION_ROUTES.WEBVIEW}?urlToLoad=hello://google.it`
    );
```
- to generate a **webview** error, turn off the dev-server before the activation of the donation flow

### CODE REVIEW

**Review by emiliopavia — APPROVED** [review]

**Comment by emiliopavia** [comment]
I'm waiting for the approval from @thisisjp in order to merge this.

**Review by thisisjp — APPROVED** [review]

**Pull request merged**
